.github/workflows/release.yml

```diff
@@ -0,0 +1,116 @@
+name: release
+on:
+  push:
+    tags:
+    - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  create-release:
+    name: create-release
+    runs-on: ubuntu-latest
+    env:
+      GITU_VERSION: ""
+    outputs:
+      upload_url: ${{ steps.release.outputs.upload_url }}
+      gitu_version: ${{ env.GITU_VERSION }}
+    steps:
+      - name: Get the release version from the tag
+        shell: bash
+        if: env.GITU_VERSION == ''
+        run: |
+          # See: https://github.community/t5/GitHub-Actions/How-to-get-just-the-tag-name/m-p/32167/highlight/true#M1027
```

Comment:
The link is broken, seems there's a `github.ref_name` / `GITHUB_REF_NAME` nowadays: https://github.com/orgs/community/discussions/26686

Reply:
updated, I've omitted the community discussion link since they don't seem to stick around